src/CLI/renderer/scripts/render_cli.py

```diff
@@ -145,6 +145,10 @@ def datetimeformat(time):
 
     j2_env.globals.update(datetimeformat=datetimeformat)
 
+    full_cmd = os.getenv('USER_COMMAND', None)
```

Comment:
Environment variable is global for Klish processes to access and overwrite it. It's good for configuration, but not suggested for this case. Can we pass the user command as a string?